components/style/context.rs

```diff
@@ -394,7 +394,10 @@ impl TraversalStatistics {
             dependency_selectors: stylist.num_invalidations() as u32,
             declarations: stylist.num_declarations() as u32,
             stylist_rebuilds: stylist.num_rebuilds() as u32,
-            traversal_time_ms: (time::precise_time_s() - start) * 1000.0,
+            traversal_time_ms: (SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap_or_default()
+            .as_secs_f64() - start) * 1000.0,
```

Comment:
Oh, sorry that I didn't notice this before, but please remove all changes from `style`. We are downstream from style which is in the Gecko source tree.